lib/elixir/lib/keyword.ex

```diff
@@ -669,7 +667,12 @@ defmodule Keyword do
   """
   @spec pop(t, key, value) :: {value, t}
   def pop(keywords, key, default \\ nil) when is_list(keywords) do
```

Comment:
@josevalim what do you think about having `get`, `pop`, `put_new` implemented as `_lazy` counterpart with `fn -> default end`?